### DISCUSSION TITLE
Trying to store bash script variables in yaml

### DISCUSSION THREAD

**Discussion by cloud8mike** [mikefarah/yq#1393] [Q&A]
I have a bash script with lots of variables and would like to use yaml rather than deal with bash variables. I can get this to work using JSON but not YAML, any ideas what I'm doing wrong?

```sh
# Works with JSON
read -r -d '' ITEMS_JSON <<- EOF
{
"one": "apple",
"two": "orange"
}
EOF
echo $ITEMS_JSON | yq .two
> orange

# Does NOT work with YAML
read -r -d '' ITEMS_YAML <<- EOF
one: apple
two: orange
EOF
echo $ITEMS_YAML | yq .one
> Error: bad file '-': yaml: mapping values are not allowed in this context
```

**Comment by mikefarah** [comment]
This example works for me - which version of yq are you using? Try downloading the latest one from releases/docker/etc
